src/psyclone/psyGen.py

```diff
@@ -5186,12 +5203,19 @@ class SymbolTable(object):
     Encapsulates the symbol table and provides methods to declare new symbols
     and look up existing symbols. It is implemented as a single scope
     symbol table (nested scopes not supported).
+
+    :param kernel: Reference to the KernelSchedule it belongs.
```

Comment:
"...KernelSchedule to which this symbol table belongs."

Reply:
done